### ISSUE TITLE
Publish new release for DMD's CI

### ISSUE THREAD

**Issue by JinShil** [msgpack/msgpack-d#102]
msgpack-d is one of two projects causing a Jenkins CI failure for DMD (https://github.com/dlang/dmd/pull/8241) and preventing us from moving forward with a deprecation.   Looking through the code in msgpack-d's master branch, it appears the fix has already been made, but it needs to be tagged and released for DMD's CI to pick it up.

**Comment by AndrejMitrovic** [comment]
Ping @repeatedly? The last non-beta release can't compile due to errors:

```
../../.dub/packages/msgpack-d-0.9.6/msgpack-d/src/msgpack.d(3670,15): Error: catch statement without an exception specification is deprecated
../../.dub/packages/msgpack-d-0.9.6/msgpack-d/src/msgpack.d(3670,15):        use catch(Throwable) for old behavior
dmd failed with exit code 1.
```

**Comment by repeatedly** [comment]
@AndrejMitrovic You want non beta version, right?

**Comment by AndrejMitrovic** [comment]
Yeah, it would be great to have. I'm not sure if I can use the beta version in dub, for example.
